pages/blueprints/index.js

```diff
@@ -43,6 +43,9 @@ const messages = defineMessages({
   errorTitle: {
     defaultMessage: "An Error Occurred"
   },
+  errorTitleInactive: {
+    defaultMessage: "Image Building Service (lorax-composer) is Not Active"
```

Comment:
This message seems a bit clunky to me. Do we need to mention that it's called `lorax-composer` in the title?

Do we usually use title case?

Reply:
Here is PF3 documentation for the [Empty State pattern](https://www.patternfly.org/pattern-library/communication/empty-state/). They use headline style capitalization. But in PF4 they are making a change to recommend sentence style for most strings. I'd opt to follow what other Cockpit pages use now—which I think is headline—and update later when we migrate to PF4.